resources/js/copy_clipboard.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file
+ * Attaches behaviors to copy text to clipboard for elements with data-copy-target attribute.
+ */
+(function (Drupal) {
+
+  Drupal.behaviors.copyClipboard = {
+    attach: function (context) {
+      // Use a scoped function to handle click events
+      const handleClick = function (event) {
+        const element = event.currentTarget;
+        const targetSelector = element.getAttribute('data-copy-target');
+        const targetElement = document.querySelector(targetSelector);
```

Comment:
I've tested this with 2 copyright patterns on the page and it always prints the content of the first one.
I used this to print it twice:
```
{{ pattern('copyright_overlay', {
    trigger_label: 'Trigger label 1',
    modal_title: 'Modal title 1',
    content: 'Content 1',
  }) }}

  {{ pattern('copyright_overlay', {
    trigger_label: 'Trigger label 2',
    modal_title: 'Modal title 2',
    content: 'Content 2',
  }) }}
```
and if I do `console.log(targetSelector);` it prints ".copyright-content" no matter which I click